extension/src/countminsketch.rs

```diff
@@ -0,0 +1,143 @@
+use pgx::*;
+
+use aggregate_builder::aggregate;
+use countminsketch::{CountMinHashFn, CountMinSketch as CountMinSketchInternal};
+use flat_serialize::*;
+
+use crate::{
+    flatten,
+    palloc::{Inner, Internal},
+    pg_type,
+    raw::bytea,
+    ron_inout_funcs,
+};
+
+#[pg_schema]
+pub mod toolkit_experimental {
+    use super::*;
+
+    pg_type! {
+        #[derive(Debug)]
+        struct CountMinSketch<'input> {
+            width: u32,
+            depth: u32,
+            hash_keys: [u64; self.depth],
```

Comment:
Given that we don't allow a user to input these, I'm not sure it's worth actually storing them.  We should be able to rely on these being the set 1..depth.